interceptor.go

```diff
@@ -78,245 +74,108 @@ func (i *Interceptor) getAndInitInstrument(isClient bool) *instruments {
 
 // WrapUnary implements otel tracing and metrics for unary handlers.
 func (i *Interceptor) WrapUnary(next connect.UnaryFunc) connect.UnaryFunc {
-	return func(ctx context.Context, request connect.AnyRequest) (connect.AnyResponse, error) {
-		requestStartTime := i.config.now()
-		req := &Request{
-			Spec:   request.Spec(),
-			Peer:   request.Peer(),
-			Header: request.Header(),
+	return func(ctx context.Context, req connect.AnyRequest) (connect.AnyResponse, error) {
+		startAt := i.config.now() // TODO: get from connect
```

Comment:
Added TODO here, the only accurate way to calculate the time would be for connect to pass the start time.